### PR TITLE
My suggestions

### PR DESCRIPTION
**Pull request by vcfxb**
See the commit note.

### CODE REVIEW

File: src/lib.rs [hunk 1]
```diff
@@ -78,20 +82,15 @@ impl From<checked_command::Error> for Error {
...
 impl error::Error for Error {
-    fn description(&self) -> &str {
-        "Process error"
-    }
-
-    fn cause(&self) -> Option<&error::Error> {
-        Some(self)
-    }
+    fn description(&self) -> &str { "Process error" }
+    fn cause(&self) -> Option<&error::Error> {Some(self)}
```
Comment (otavio) [inline src/lib.rs:86]: This is the rustfmt style so it is not "mine".

**Review by otavio — CHANGES_REQUESTED** [review]
Thanks for all your changes. 

Is it possible for you to remove the changes comments and update the pull request?

Not sure how you usually do but, if possible, split the commit so we can have proper commit messages on each modification.

File: src/lib.rs (hunk 1 above)
Comment (vcfxb) [reply src/lib.rs:86]: Sounds good to me. I don't use rustfmt, so I wouldn't know, but neither way is wrong. This is mostly a preference thing.

**Review by vcfxb — COMMENTED** [review]
The change there is mostly a style thing, so it looks good to me.

**Pull request closed without merging**